shared/chat/conversation/messages/wrapper/container-wrapper-author.js

```diff
@@ -53,29 +52,18 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
 const mergeProps = (stateProps, dispatchProps, {measure}) => {
   const {message, previous} = stateProps
 
-  const continuingTextBlock =
+  const sequentialUserMessages =
     previous &&
     previous.author === message.author &&
-    (previous.type === 'text' || previous.type === 'deleted')
-
-  const oldEnough = !!(
-    previous &&
-    previous.timestamp &&
-    message.timestamp &&
-    message.timestamp - previous.timestamp > howLongBetweenTimestampsMs
-  )
+    (previous.type === 'text' || previous.type === 'deleted' || previous.type === 'attachment')
 
-  // Always show a timestamp if the previous message is a concise joined/left message
-  const previousIsJoinedLeft = !!(
-    previous &&
-    (previous.type === 'systemJoined' || previous.type === 'systemLeft')
-  )
+  const oldEnough = Constants.enoughTimeBetweenMessages(message, previous)
 
   const timestamp =
-    stateProps.orangeLineAbove || !previous || oldEnough || previousIsJoinedLeft
-      ? formatTimeForMessages(message.timestamp)
-      : null
-  const includeHeader = !previous || !continuingTextBlock || !!timestamp
+    stateProps.lastPositionExists || !previous || oldEnough ? formatTimeForMessages(message.timestamp) : null
```

Comment:
why actually build the timestamp w/ formatTimeForMessages? we just need the boolean right

Reply:
`SendIndicator` takes the timestamp from `props.message.timestamp`, which is the `number`. But it shouldn't be reaching into the `message` to grab that, so you should pass in `message.timestamp` directly as `timestamp` and use that for `SendIndicator` instead